### ISSUE TITLE
Feature request: Subcommand aliases

### ISSUE THREAD

**Issue by ayakovlenko** [scallop/scallop#118]

**Comment by Rogach** [comment]
Yes, that can be done. Will the following syntax be okay?

``` scala
val sub = new Subcommand(List("cmd", "alias1", "alias2")) {
}
addSubcommand(sub)
```

**Comment by ayakovlenko** [comment]
I think varargs constructor looks even better:

``` scala
val sub = new Subcommand("cmd", "alias1", "alias2") {
}
addSubcommand(sub)
```

**Comment by Rogach** [comment]
Yes, that's good. I'll implement it shortly.

**Comment by Rogach** [comment]
Released `2.0.3` with the change, should be available withing an hour.

Please test it, and I'll close issue if everything is OK.

**Comment by ayakovlenko** [comment]
Works great. Thank you!
